bottlecap/Cargo.lock

```diff
@@ -80,12 +80,6 @@ dependencies = [
  "bytemuck",
 ]
 
-[[package]]
```

Comment:
Nice to see all these dependencies going

Reply:
Yeah!
This was also causing errors on compiling